10_files_lab/fileslab.txt

```diff
@@ -1,37 +1,122 @@
-Now create a new directory named "myinfo" and move the file into it.
-
-Did it work? Why? Can you create a file "myinfo" inside a directory named "myinfo" ? How ?
-
-5. Move the files "main.c", "game.c", "enemy.c" and "hero.c" to folder "Music".
-
-Write here the commands you used:
-
-6. Move the folder "Music" with all its contents into directory "Drivers"
-
-Write here the commands you used:
-
-
-7. Try to use "rmdir" to delete directory "Drivers". Did it work? Why? How can you delete directory "Drivers" ?
+Files and Directories Lab
+Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
+
+1. Create a new folder named "lab" in your home directory and inside create the files:
+
+  - a.out, hero.c, enemy.c, game.c, main.c
+  - human.h, monster.h
+  - .highscore
+
+Write here the commands you used:
+
+mkdir lab
+touch a.out hero.c enemy.c games.c main.c human.h monster.h .highscore
+
+maxim@maxim-virtual-machine:~/lab$ ls -l
+total 0
+-rw-rw-r-- 1 maxim maxim 0 Dec 21 11:52 a.out
+-rw-rw-r-- 1 maxim maxim 0 Dec 21 11:52 enemy.c
+-rw-rw-r-- 1 maxim maxim 0 Dec 21 11:52 games.c
+-rw-rw-r-- 1 maxim maxim 0 Dec 21 11:52 hero.c
+-rw-rw-r-- 1 maxim maxim 0 Dec 21 11:52 human.h
+-rw-rw-r-- 1 maxim maxim 0 Dec 21 11:52 main.c
+-rw-rw-r-- 1 maxim maxim 0 Dec 21 11:52 monster.h
+maxim@maxim-virtual-machine:~/lab$ ls -lha
+total 8.0K
+drwxrwxr-x  2 maxim maxim 4.0K Dec 21 11:52 .
+drwxr-xr-x 17 maxim maxim 4.0K Dec 21 10:09 ..
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 a.out
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 enemy.c
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 games.c
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 hero.c
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 .highscore
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 human.h
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 main.c
+-rw-rw-r--  1 maxim maxim    0 Dec 21 11:52 monster.h
+
+2. "cd" into lab folder and inside create the following subdirectories:
+  - Music
+  - Misc
+  - Drivers
+
+Write here the commands you used:
+
+maxim@maxim-virtual-machine:~/lab$ mkdir Music Misc Drivers
+maxim@maxim-virtual-machine:~/lab$ ls -l
+total 12
+-rw-rw-r-- 1 maxim maxim    0 Dec 21 11:52 a.out
+drwxrwxr-x 2 maxim maxim 4096 Dec 21 11:54 Drivers
+-rw-rw-r-- 1 maxim maxim    0 Dec 21 11:52 enemy.c
+-rw-rw-r-- 1 maxim maxim    0 Dec 21 11:52 games.c
+-rw-rw-r-- 1 maxim maxim    0 Dec 21 11:52 hero.c
+-rw-rw-r-- 1 maxim maxim    0 Dec 21 11:52 human.h
+-rw-rw-r-- 1 maxim maxim    0 Dec 21 11:52 main.c
+drwxrwxr-x 2 maxim maxim 4096 Dec 21 11:54 Misc
+-rw-rw-r-- 1 maxim maxim    0 Dec 21 11:52 monster.h
+drwxrwxr-x 2 maxim maxim 4096 Dec 21 11:54 Music
+
+
+3. Copy the file "/etc/passwd" to each subdirectory created in (2).
+
+Write here the commands you used:
+
+maxim@maxim-virtual-machine:~/lab$ cp /etc/passwd /home/maxim/lab/Drivers/ &&  cp /etc/passwd /home/maxim/lab/Misc/ &&  cp /etc/passwd /home/maxim/lab/Music/

```

Comment:
correct!

but consider using relative paths here.. (./lab/...)
or the shorthand for home (~): ```~/ # same as /home/maxin/```

also you can:
```
for dest in "Drivers Misc Music"; 
    do cp /etc/passwd "~/lab/$dest/"; 
done
```

:)